2020/OSCON/innersource-open-inside-the-firewall/outline.md

```diff
@@ -0,0 +1,43 @@
+# Title
+
+InnerSource: Open Inside the Firewall
+
+# Abstract
+
+Open source helps create the world's most impactful software.
+It allows people from all walks to collaborate on projects that fill a common need.
+InnerSource applies the lessons learned from open source to company-internal software development.
+Sustained InnerSource practice unlocks developer creativity and efficiency.
+It reduces siloed, duplicated work and creates a dynamic, fulfilling work environment.
+
+This talk gives an introduction and overview of the breadth of current state of InnerSource in the industry.
+We'll discuss motivations for InnerSource, benefits seen, challenges faced, and areas of active investigation.
+This content comes not just from a single perspective,
+but from the breadth of experiences shared by dozens of software companies as told in the InnerSource Commons,
+an open organization dedicated to education and spread of InnerSource.
+
+No matter what your current interest or motivation for InnerSource, this presentation will give you the latest experiences that others in your place have already had.
+You'll be ready to build on their learnings.
+You'll also learn how to leverage the InnerSource Commons to stay in touch on an ongoing basis with the latest industry experiences in InnerSource.
+
+# Audience
+
+People that already believe that open source is the right way to develop software.
+
+# Takeaways
+
+* InnerSource is the application of open source methodology to company-internal software development.
```

Comment:
I'm not sure if open source is a  well delimited, formal methodology, comparable to e.g. Scrum.
One of definitions in the usual academic papers in this context has this: "Inner Source (IS) is the use of open source software development practices and the establishment of an open source-like culture within organizations."
Might be to verbose for a short take-away sentence though.

Other than that: I agree with the take-away messages. :+1:

Reply:
Good point.  Updated.